OrbitQt/main.cpp

```diff
@@ -8,11 +8,16 @@
 
 #include "../OrbitGl/App.h"
 #include "orbitmainwindow.h"
+#include "CrashHandler.h"
+#include "Path.h"
 
 int main(int argc, char* argv[]) {
 #if __linux__
   QCoreApplication::setAttribute(Qt::AA_DontUseNativeDialogs);
 #endif
+#ifdef _WIN32
+  CrashHandler m_CrashHandler(s2ws(Path::GetDumpPath().c_str()));
```

Comment:
does it have to be wstring? Please try to avoid them if possible.

Reply:
CrashpadClient requires paths to database and handler to be passed as base::FilePath which accepts only std::wstring (or another base::FilePath) as an argument to the constructor.